app/assets/config/manifest.js

```diff
@@ -0,0 +1,3 @@
+//= link_tree ../images
```

Comment:
I think this is something brought by Rails 5.1

Again, we'll wait the stable version to be released before teaching it.